### PR TITLE
breakdown cli quickstart steps 

### PR DESCRIPTION
**Pull request by mirnawong1**
the core quickstart currently embeds multiple h3 steps into one h2. this makes the pages quite long and misrepresents the steps as well. 

After discussion with @runleonarun and feedback from @greg-mckeon , this pr breaks down the h3 steps into h2 to give users more bite-sized chunks of content.

this pr also adds redirects from steps to anchored links.

[docs project](https://www.notion.so/dbtlabs/Breakdown-Core-quickstart-into-steps-0f1d500635fa4d288371dc4a033cc4e4)

### CODE REVIEW

File: website/static/_redirects [hunk 1]
```diff
@@ -1,3 +1,11 @@
+
+# Redirects for quickstarts sections that were split
+
+/quickstarts/manual-install?step=2 /quickstarts/manual-install?step=2 301
```
Comment (runleonarun) [inline website/static/_redirects]: @mirnawong1 I'm going to remove the redirects so it won't affect future links to the correct steps. Also, not sure we can do redirects at this level
```suggestion
```

File: website/static/_redirects [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+/quickstarts/manual-install?step=3 /quickstarts/manual-install?step=7 301
```
Comment (runleonarun) [inline website/static/_redirects]:
```suggestion
```

File: website/static/_redirects [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+/quickstarts/manual-install?step=4 /quickstarts/manual-install?step=13 301
```
Comment (runleonarun) [inline website/static/_redirects]:
```suggestion
```

File: website/static/_redirects [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
+/quickstarts/manual-install?step=5 /quickstarts/manual-install?step=15 301
```
Comment (runleonarun) [inline website/static/_redirects]:
```suggestion
```

File: website/static/_redirects [hunk 5]
```diff
@@ -1,3 +1,7 @@
+
+# Redirects for quickstarts sections that were split
```
Comment (runleonarun) [inline website/static/_redirects]:
```suggestion
```

File: website/static/_redirects [hunk 6]
```diff
@@ -1,3 +1,6 @@
+
```
Comment (runleonarun) [inline website/static/_redirects]:
```suggestion
```

File: website/static/_redirects [hunk 7]
```diff
@@ -1,3 +1,6 @@
...
+
```
Comment (runleonarun) [inline website/static/_redirects]:
```suggestion
```

File: website/static/_redirects [hunk 8]
```diff
@@ -1,3 +1,6 @@
...
+
```
Comment (runleonarun) [inline website/static/_redirects]:
```suggestion
```

File: website/static/_redirects (hunk 1 above)
Comment (mirnawong1) [reply website/static/_redirects]: awesome! should we do a crawler trigger after this pr is merged @runleonarun ? just in case

Comment (runleonarun) [reply website/static/_redirects]: @mirnawong1 probably a good idea. I can try out our new process!

**Review by runleonarun — APPROVED** [review]
Looks good once John helps me figure out why I'm getting an error when I refresh the page.

**Pull request merged**